org.idempiere.test/src/org/idempiere/test/model/MappedModelFactoryTest.java

```diff
@@ -65,8 +65,10 @@ public MappedModelFactoryTest() {
 	@Order(1)
 	public void testDefaultMappedModelFactory() {
 		IMappedModelFactory mappedFactory = Core.getMappedModelFactory();
-		mappedFactory.addMapping(MyTest.Table_Name, () -> MyTest.class, (id, trxName) -> new MyTest(Env.getCtx(), id, trxName), 
-				(rs, trxName) -> new MyTest(Env.getCtx(), rs, trxName));		
+		mappedFactory.addMapping(MyTest.Table_Name, () -> MyTest.class, 
+		                         (id, trxName) -> new MyTest(Env.getCtx(), id, trxName), 
+		                         (rs, trxName) -> new MyTest(Env.getCtx(), rs, trxName), 
+		                         (uuID, trxName) -> new MyTest(Env.getCtx(), uuID, trxName));
```

Comment:
please revert the changes to this class

Reply:
@hengsin I tried resetting but not getting any changes uncommitted.